quizara-frontend/src/reducers/QuizReducer.tsx

```diff
@@ -31,12 +30,6 @@ const Quizreducer = (state: InitialState, action: Action) => {
         },
       };
 
-
-      // width: 70%;
-      // padding: 1rem;
-      // margin: 0.5rem;
-
-
     case "TOGGLE_ISOPTIONSENABLED":
```

Comment:
Every word should be separated by underscore while naming actions i.e `TOGGLE_IS_OPTIONS_ENABLED`, so that it is readable.